setup.cfg

```diff
@@ -14,11 +14,12 @@ exclude =
 # D107 requires docstrings on __init__ but these are ignored in our current sphinx configuration
 # RST304 is about "unknown text roles" in sphinx. Unfortunately, things like :class: appear to be unknown
 # RST902 is from trouble parsing the __all__ entry. This only happens in a single file
+# D413 requires an empty line at the end of every docstring.
```

Comment:
Why exclude it? Because it adds another 15 files that I didn't mean to touch.

```
.\fairlearn\datasets\_fetch_boston.py:14:1: D413 Missing blank line after last section
.\fairlearn\datasets\_fetch_boston.py:14:1: D413 Missing blank line after last section
.\fairlearn\metrics\_extra_metrics.py:22:1: D413 Missing blank line after last section
.\fairlearn\metrics\_extra_metrics.py:22:1: D413 Missing blank line after last section
.\fairlearn\metrics\_extra_metrics.py:80:1: D413 Missing blank line after last section
.\fairlearn\metrics\_extra_metrics.py:80:1: D413 Missing blank line after last section
.\fairlearn\metrics\_extra_metrics.py:111:1: D413 Missing blank line after last section
.\fairlearn\metrics\_extra_metrics.py:111:1: D413 Missing blank line after last section
.\fairlearn\metrics\_extra_metrics.py:142:1: D413 Missing blank line after last section
.\fairlearn\metrics\_extra_metrics.py:142:1: D413 Missing blank line after last section
.\fairlearn\metrics\_extra_metrics.py:173:1: D413 Missing blank line after last section
.\fairlearn\metrics\_extra_metrics.py:173:1: D413 Missing blank line after last section
.\fairlearn\postprocessing\_threshold_optimizer.py:94:1: D413 Missing blank line after last section
.\fairlearn\postprocessing\_threshold_optimizer.py:94:1: D413 Missing blank line after last section
.\fairlearn\postprocessing\_tradeoff_curve_utilities.py:38:1: D413 Missing blank line after last section
.\fairlearn\postprocessing\_tradeoff_curve_utilities.py:38:1: D413 Missing blank line after last section
.\fairlearn\postprocessing\_tradeoff_curve_utilities.py:71:1: D413 Missing blank line after last section
.\fairlearn\postprocessing\_tradeoff_curve_utilities.py:71:1: D413 Missing blank line after last section
.\fairlearn\reductions\_exponentiated_gradient\exponentiated_gradient.py:20:1: D413 Missing blank line after last section
.\fairlearn\reductions\_exponentiated_gradient\exponentiated_gradient.py:20:1: D413 Missing blank line after last section
.\fairlearn\reductions\_exponentiated_gradient\exponentiated_gradient.py:65:1: D413 Missing blank line after last section
.\fairlearn\reductions\_exponentiated_gradient\exponentiated_gradient.py:65:1: D413 Missing blank line after last section
.\fairlearn\reductions\_exponentiated_gradient\exponentiated_gradient.py:194:1: D413 Missing blank line after last section
.\fairlearn\reductions\_exponentiated_gradient\exponentiated_gradient.py:194:1: D413 Missing blank line after last section
.\fairlearn\reductions\_exponentiated_gradient\exponentiated_gradient.py:228:1: D413 Missing blank line after last section
.\fairlearn\reductions\_exponentiated_gradient\exponentiated_gradient.py:228:1: D413 Missing blank line after last section

Reply:
Must be some kind of new flake8 rule?

Reply:
Odd that it hasn't blown up our other builds.